kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/SchemaManager.java

```diff
@@ -147,6 +154,7 @@ private SchemaManager(
     this.tableCreateLocks = tableCreateLocks;
     this.tableUpdateLocks = tableUpdateLocks;
     this.schemaCache = schemaCache;
+    this.cache = cache;
```

Comment:
If the cache is going to be accessed from the `SchemaManager`, we need to either make it thread-safe, or manually synchronize access to it, since the `SchemaManager` and the `BigQuerySinkTask` might access it at the same time.